app/models/UserStore.ts

```diff
@@ -115,8 +119,14 @@ export const UserStoreModel = types
     get getMetadata() {
       return self.metadata
     },
-    get getPrivMesages() {
-      return [...new Map(self.privMessages.slice().map((item) => [item.pubkey, item])).values()]
+    get getChats() {
+      const chats = getSnapshot(self.privMessages)
+      chats.forEach((chat) => {
+        if (chat.pubkey === self.pubkey) {
+          chat.pubkey = chat.tags.find((el) => el[0] === "p")[1]
```

Comment:
the tag is the recipient, pubkey is sender.

now you don't know who sent the message

no way this can render right

Reply:
it render fine, it made it for this case
(Home Screen)
```
current user -> send message to A -> A not response -> last message display in home screen is current user
current user -> send message to B -> B not response -> last message display in home screen is current user too
```
so I made it get recipient for this case to prevent duplicate items in home screen, and ignore the rest
message's content will not change, this behavior similar to telegram too

<img width="373" alt="Screenshot 2023-07-14 at 21 34 55" src="https://github.com/ArcadeLabsInc/arcade/assets/123083837/b732ee75-7bca-4328-921c-ddc68284bb64">